src/Hazelcast.Net.Examples/DistributedObjects/MapLockExample.cs

```diff
@@ -66,9 +65,10 @@ public static async Task Main(string[] args)
             // simply running the code here would break the example, as it would run
             // with the same context, i.e. the context that has the lock, so we need
             // to start it with a new context
-            //
-            var task = AsyncContext.RunWithNew(async () =>
+
```

Comment:
I need to update our online documentation about `AsyncContext`. but essentially, we recommend the following:
```
Task task;
using (AsyncContext.New())
{
    task = Task.Run(...);
}
```

Alternatively,
```
var task = Task.Run(async () => 
{
    AsyncContext.RequireNew();
    // ...
});
```

Can you replace with the 1st syntax? The one you are using *is* working but... I'd rather avoid it as it can lead to some confusion.

Reply:
Sure, fixed at https://github.com/hazelcast/hazelcast-csharp-client/pull/820/commits/869e40b921a08fbb27fc387bffcc08564cb430d9